chemprop/nn/predictors.py

```diff
@@ -94,7 +94,7 @@ def __init__(
         self.ffn = MLP.build(
             input_dim, n_tasks * self.n_targets, hidden_dim, n_layers, dropout, activation
         )
-        self.criterion = criterion or self._default_criterion
+        self.criterion = criterion or self._default_criterion(w_t=torch.ones(self.n_tasks))
```

Comment:
how does a user not specify their loss function but _do_ want to specify task weights (i.e., they want to use the default for the predictor but have a multi-task dataset where some tasks are more important)?

Reply:
My thinking was the default criterion for an e.g. `RegressionFFN` is `MSELoss` with all weights equal to 1. An `MSELoss` with different task weights is not the default and therefore needs to be custom created `criterion=MSELoss(w_t=...)` and given to the predictor.

Reply:
that seems slightly cumbersome, as currently we allow a user to specify _just_ task weights. Ideally, we should keep this behavior.

Reply:
An alternative is to have `_FFNPredictorBase` optionally take `w_t` as an argument along with `criterion`. Then `w_t` could be specified when making the predictor without needing to specify the criterion. The main reason I soft against this is that this adds a bit of complexity to the call signature of predictors to make what is already possible (custom `w_t`) slightly easier. Only soft against it because it isn't a bit change and I don't have a gauge for how often someone will want to modify the task weights.